core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -181,7 +181,7 @@
 		'protected'                   => 'groups',
 		'createSitemap'               => 'sitemapName',
 		'includeLayout'               => 'layout',
-		'includeCache'                => 'cache,clientCache',
+		'includeCache'                => 'cache,alwaysLoadFromCache,clientCache',
```

Comment:
Adding the checkbox after both dropdowns gives a better look&feel imho

Reply:
I don't think that's true because the checkbox belongs to the shared cache only. This is how it looks now:

<img width="1190" alt="Bildschirmfoto 2019-06-03 um 13 05 00" src="https://user-images.githubusercontent.com/481937/58797402-36dbb080-8600-11e9-8d16-4038a65643c3.png">

Reply:
Haven't we agreed on not having a checkbox at all?

Reply:
AFAIR we have agreed not to add the checkbox, because we cannot foresee which problems it will cause (although we can foresee that it will make everything a lot more complex). Do we need to rediscuss?

Reply:
It's funny how people always keep saying "we agreed". Maybe things we discuss on Mumble need to be documented better because the way I remember it is that we really do want to have this checkbox and if we're able to find a solution that is fairly simple to understand and implement, we'll do it. Only if there's no simple way to get it working, we'll remove it. I then said I'll have another look at it which I did and this is the result.

I cannot imagine any side effects other than people **not** getting personal content if they should. This would be a misconfiguration then and is not relevant for security. The other way around (people getting personal content of others) would be security relevant but this not possible and remains unaffected by this checkbox.

So I would definitely have it merged the way it is so @ausi can go ahead and test his use case and see if it even works with the remember me (which I'm confident it will together with #483 ) 😄

Reply:
@aschempp @ausi What do you recall?

Reply:
We discussed to not add it because there was no easy solution. But then we found one, so this implementation certainly is right 🙃

Reply:
That's great. Then you should probably share your solution with us, so we can discuss it altogether.

Reply:
The solution is already implemented 😂 it's the one-pixel image request, which will fetch the cookies.

Reply:
> the way I remember it is that we really do want to have this checkbox and if we're able to find a solution that is fairly simple to understand and implement, we'll do it.

Yes, AFAIR this was the latest state of our discussion.